lib/NavLink/NavLink.js

```diff
@@ -1,47 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
-import NavListItem from '@folio/stripes-components/lib/NavListItem';
 import ServicePointsModal from '../ServicePointsModal';
 
 class NavLink extends React.Component {
```

Comment:
Of course this shouldn't really be called NavLink any more.

Reply:
Good catch! I will clean it up.